runner/utils/driver/istio/get.go

```diff
@@ -316,3 +316,49 @@ func (i *Istio) GetServiceRoutes(ctx context.Context, projectID string) (map[str
 
 	return serviceRoutes, nil
 }
+
+// GetServiceRole gets the service role rules of each service
+func (i *Istio) GetServiceRole(ctx context.Context, projectID string) (map[string]model.Role, error) {
+	ns := projectID
+
+	rolelist, err1 := i.kube.RbacV1().Roles(ns).List(ctx, metav1.ListOptions{})
+
+	clusterRoleList, err2 := i.kube.RbacV1().ClusterRoles().List(ctx, metav1.ListOptions{})
```

Comment:
Add a label selector in listOptions{} struct.
"app.kubernetes.io/managed-by=space-cloud"
And handle error seperately